Doppler.ImageAnalyzer.Api/Services/MongoDB/ImageAnalyzerMongoDBContextExtensions.cs

```diff
@@ -15,11 +16,11 @@ public static IServiceCollection AddImageAnalyzerMongoDBContext(this IServiceCol
             var imageAnalizerMongoDBContextSettings = new ImageAnalyzerMongoDBContextSettings();
             mongoDBContextSettingsSection.Bind(imageAnalizerMongoDBContextSettings);
 
-            var mongoUrlBuilder = new MongoUrlBuilder(imageAnalizerMongoDBContextSettings.ConnectionString);
-
-            // when these values were not specified in the "ConnectionString"
-            mongoUrlBuilder.DatabaseName ??= imageAnalizerMongoDBContextSettings.DatabaseName;
-            mongoUrlBuilder.Password ??= imageAnalizerMongoDBContextSettings.Password;
+            var mongoUrlBuilder = new MongoUrlBuilder(imageAnalizerMongoDBContextSettings.ConnectionString)
+            {
+                DatabaseName = imageAnalizerMongoDBContextSettings.DatabaseName,
+                Password = imageAnalizerMongoDBContextSettings.Password,
+            };
```

Comment:
I believe that with this change, when the database name and/or password are part of the connection string, they will always be overwritten with the configuration class value, even when no value is defined in the configuration class and remains null

The previous code, in my opinion, contemplates the scenario of establishing those two values only by connection string

Could you check this?

Reply:
It is being considered [here](https://github.com/FromDoppler/doppler-image-analyzer/blob/bdf6de63848c2e391057d427f38dca2804886e64/Doppler.ImageAnalyzer.Api/Services/MongoDB/ImageAnalyzerMongoDBContextSettings.cs#L15), when the **database name** (or **password**) is defined in the **connection string**, these values are not overwritten.

The idea to solve this in the `ImageAnalyzerMongoDBContextSettings` file was to use the database, justly, defined in the connection string, when trying to obtain the proper collection in each service (only one for now), for example [here](https://github.com/FromDoppler/doppler-image-analyzer/blob/bdf6de63848c2e391057d427f38dca2804886e64/Doppler.ImageAnalyzer.Api/Services/MongoDB/ImageAnalyzerMongoDBContextSettings.cs#L15) (in [similar way it's solved in the push-contacts](https://github.com/FromDoppler/doppler-push-contact/blob/main/Doppler.PushContact/Services/PushContactService.cs#L594) project)